_component_design/tables-nested.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Tables - Nested
```

Comment:
File name can stay the same, but the title should be "Nested Tables"

Reply:
updated.